src/main/nativeMessaging.main.ts

```diff
@@ -19,15 +20,16 @@ export class NativeMessagingMain {
 
         ipc.serve(() => {
             ipc.server.on('message', (data: any, socket: any) => {
-                // This is a ugly hack until electron is updated 7.0.0 which supports ipcMain.invoke
```

Comment:
Does it make sense to remove this comment now?

Reply:
Yes, the implementation was changed slightly in order to support sending multiple messages to the browser instead of only responses. Which is needed to let the browser know we want to validate the fingerprint.